x-pack/plugins/cases/public/api/decoders.ts

```diff
@@ -41,12 +39,11 @@ export const decodeCasesMetricsResponse = (metrics?: CasesMetricsResponse) =>
     fold(throwErrors(createToasterPlainError), identity)
   );
 
-export const decodeCasesBulkGetResponse = <Field extends keyof Case = keyof Case>(
-  res: CasesBulkGetResponseCertainFields<Field>,
-  fields?: string[]
-) => {
-  const typeToDecode = getTypeForCertainFieldsFromArray(CasesRt, fields);
-  pipe(typeToDecode.decode(res.cases), fold(throwErrors(createToasterPlainError), identity));
+export const decodeCasesBulkGetResponse = (res: CasesBulkGetResponse) => {
+  pipe(
+    CasesBulkGetResponseRt.props.cases.decode(res.cases),
```

Comment:
Do we want to use the entire `CasesBulkGetResponseRt` for this? We probably want to know if the errors are formatted correctly right?

Reply:
Correct! Before it was not possible because I was constructing the type dynamically.